include/dsn/dist/replication/replication.codes.h

```diff
@@ -34,6 +34,7 @@ DEFINE_THREAD_POOL_CODE(THREAD_POOL_REPLICATION)
 DEFINE_THREAD_POOL_CODE(THREAD_POOL_LOCAL_APP)
 DEFINE_THREAD_POOL_CODE(THREAD_POOL_REPLICATION_LONG)
 DEFINE_THREAD_POOL_CODE(THREAD_POOL_COMPACT)
+DEFINE_THREAD_POOL_CODE(THREAD_POOL_INGESTION)
```

Comment:
Yes, I have considered it, I meant to discuss it in next pull request in pegasus.
When replica is during ingestion, it will reject other write requests. It is better for us to shorten ingestion time. So I add a new thread pool to execute it with 24 worker count (same as replication thread pool). 
Default thread pool are shared with many functions, and some are tasks execute periodically, and worker count is 8. If I reuse it, I will add worker count for this thread pool and test for it.
If adding a new thread pool won't cost much, it is better to add a new one.